doc/results_format.json

```diff
@@ -0,0 +1,100 @@
+{
+    "data":
+    {
+        "record":
+        {
+            "total_balance_due": 199.99,
+            "cases":
+            [
+                {
+                    "name": "Doe, John",
```

Comment:
Should we move the name and birth year to the record object since they repeat in each of the objects in the cases array?

Reply:
It's possible an alias is used, hence we can't assume they repeat for each case. I haven't taken a look at the front end requirements in a while so as your working on this let me know if you find anything that is missing.

Reply:
Here's a mockup of what grouping by cases would look like:

![group-by-cases](https://user-images.githubusercontent.com/6887982/60763034-b524db80-a020-11e9-81aa-931d74e60c5c.png)

Reply:
hmmm. not sure. It would be best to get Michaels input... I know one thing for sure he wants is to have all eligible cases grouped together. Of course there could probably be a situation where a case has both eligible and ineligible charges. 

Perhaps what he means is to groups all cases that have eligible charges at the top so that the person can quickly navigate to cases that have eligible charges...

Even if we show Michael this he could change his mind after seeing it and using it...

Reply:
Yeah, I'll ask Michael about not requiring the fields.

A somewhat related question - if a search/analysis is preformed that returns multiple people (say only first name and birth date is submitted), is it correct that the time eligibility would look at the records/cases as a whole?

If so, we wouldn't get an accurate time eligibility since it's considering more than one person.

Reply:
Yes, if the results returned contain multiple people it will treat them all as belonging to the same person. However, Michael said that they would only put in search parameter's that will return a single person. Also the system requires first and last name, but you can use `*` filter to get more results such as first-name: `nic*` and last-name: `sch*`... Although we don't have to worry about that, because they wont be using it like that.

Reply:
Got it, thanks Nick!

Reply:
Michael said "Yes that would definitely be more helpful" to group by cases. I'll create some markup to support this and update the designs.